README.md

```diff
@@ -195,10 +195,11 @@ function getToken (req) {
 }
 ```
 
-We must use the `preHandler` lifecycle if we want to accept CSRF tokens
-inside request bodies. The default property matched is `_csrf`, but it's possible
-to constumize this by chainging the `getToken` function.
-If we are using headers, using the `onRequest` lifecycle is sufficient and safer.
+It is recommended to provide a custom `getToken` function for security and performance reasons.
```

Comment:
I would link some OWASP documentation like 

https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#use-of-custom-request-headers

Reply:
Like this?

```suggestion
It is recommended to provide a custom `getToken` function for performancs and [security](https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#use-of-custom-request-headers) reasons.
```

Reply:
@Eomm

Reply:
Done